packages/editor/src/lib/editor/resize.ts

```diff
@@ -0,0 +1,243 @@
+import { TLShape, TLShapeId } from '@tldraw/tlschema'
+import { Box } from '../primitives/Box'
+import { Mat, MatLike } from '../primitives/Mat'
+import { Vec, VecLike } from '../primitives/Vec'
+import { approximately, areAnglesCompatible } from '../primitives/utils'
+import { applyPartialToShape } from '../utils/applyPartialToShape'
+import { Editor, TLResizeShapeOptions } from './Editor'
+
+/**
+ * Get the partial for a resized shape.
+ *
+ * @param id - The id of the shape to resize.
+ * @param scale - The scale factor to apply to the shape.
+ * @param options - Additional options.
+ *
+ * @public
```

Comment:
this feels like an implementation detail to me, should we keep it internal for now?